### DISCUSSION TITLE
SNZB-02 new firmware

### DISCUSSION THREAD

**Discussion by JC38** [Koenkk/zigbee2mqtt#25380] [Devices]
Hello,

It seems this device got a new firmware published [https://forum.ewelink.cc/t/sonoff-snzb-02d-update-fails/101372](url) [https://forum.ewelink.cc/t/zigbee-snzb-02d-update-firmware/102657](url)
As I have many problem with some (they are sending out-of-date values, data sent is not the one displayed), I would like to test this newer firmware. Does someone know how to get it from Z2m ?

**Comment by chpego** [comment]
[In 2.1.4 version](https://github.com/Koenkk/zigbee2mqtt/pull/26565)

**Comment by JC38** [comment]
Thanks for the info.
Even if I solved my pb buying a SonOff Hub just for making these upgrades.
Never too late for the other users.
